plugin-server/src/cdp/consumers/cdp-cyclotron-delay.consumer.ts

```diff
@@ -0,0 +1,33 @@
+import { logger } from '~/utils/logger'
+
+import { Hub } from '../../types'
+import { CyclotronJobQueue } from '../services/job-queue/job-queue'
+import { CyclotronJobInvocation, CyclotronJobInvocationResult } from '../types'
+import { CdpCyclotronWorker } from './cdp-cyclotron-worker.consumer'
+
+/**
+ * Consumer for delayed invocations
+ */
+export class CdpCyclotronDelayConsumer extends CdpCyclotronWorker {
```

Comment:
Im not sure why you are extending the worker tbh - you need basically none of its functionality as it is so different to the rest of it. Feels like this would just be cleaner as its own class